activestorage/db/migrate/20170806125915_create_active_storage_tables.rb

```diff
@@ -20,6 +20,7 @@ def change
       t.datetime :created_at, null: false
 
       t.index [ :record_type, :record_id, :name, :blob_id ], name: "index_active_storage_attachments_uniqueness", unique: true
+      t.foreign_key :active_storage_blobs, column: :blob_id
```

Comment:
@georgeclaghorn I'm wondering how users that migrate from `5.2` to `6.0` will know about this change. Maybe we should create a new migration for that or/and describe what users should do in order to get this change in http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-5-2-to-rails-6-0

Reply:
👍, or we can perhaps copy an upgrade migration on `rails app:update`.

Reply:
On second thought, adding a migration on update would be nice.